modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -610,6 +618,12 @@ void ParallelCoordinates::drawLines(size2_t size) {
         drawObject.draw(i);
     }
 
+    lineShader_.setUniform("hovered", 1);
+    lineShader_.setUniform("selected", 0);
+    lineShader_.setUniform("filtered", 0);
+    if (!brushingAndLinking_.isFiltered(hoveredLine_) && hoveredLine_ != -1)
```

Comment:
if you change the order in this if statement we do not need to do the lookup when hoveredLine_ == -1
```
if (hoveredLine_ != -1 && !brushingAndLinking_.isFiltered(hoveredLine_)){
   ...
}
```